src/FacebookLoginRequestGrant.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Danjdewhurst\PassportFacebookLogin;
 
+use Laravel\Passport\Passport;
 use Illuminate\Http\Request;
 use Laravel\Passport\Bridge\User;
 use Laravel\Passport\Passport;
```

Comment:
This is already imported?

https://github.com/danjdewhurst/Laravel-Passport-Facebook-Login/pull/10/files#diff-5891a0e6c9ff01d9dc9e8713a053d607R8